### PR TITLE
Adds American Paratroopers

### PR DESCRIPTION
**Pull request by Iamgoofball**
Americans can now recieve their faction bonus of Paratroopers from their ACCs.

### CODE REVIEW

File: rules/allied-infantry.yaml [hunk 1]
```diff
@@ -120,12 +120,18 @@ e1:
...
+		UpgradeTypes: parachute
+		UpgradeMaxEnabledLevel: 0
 	Voiced:
 		VoiceSet: GIVoice
 	ProducibleWithLevel:
 		Prerequisites: barracks.infiltrated
 	QuantizeFacingsFromSequence:
 		Sequence: stand
+	WithSpriteBody@PARACHUTE:
+ 		Sequence: paradrop
```
Comment (abcdefg30) [inline rules/allied-infantry.yaml]: Looks like a space sneaked in here.

**Comment by Mailaender** [comment]
Works as promised :white_check_mark: 

image

**Comment by MustaphaTR** [comment]
Cameo is wrong as i can see from Screenshot. It needs to say Airborne which is `aparicon` iirc.

And why parachutes are not remapped?

**Comment by Phrohdoh** [comment]
:-1: currently due to the points that @MustaphaTR raises.
This "works" as in actors are paradropped, but it is not complete.

**Comment by abcdefg30** [comment]
> Cameo is wrong as i can see from Screenshot. It needs to say Airborne which is  aparicon  iirc.

I'm not sure about that, but I think there are two icons: One for the paratroopers you get from capturing airfields ("Paradrop") and another one for the american paratroopers ("Airborne").

**Comment by GraionDilach** [comment]
>And why parachutes are not remapped?

Apparently that's a limitation of `WithParachute`'s in upstream.

**Comment by Phrohdoh** [comment]
Let's get that fixed then.
[Upstream issue](https://github.com/OpenRA/OpenRA/issues/10503).

**Comment by Iamgoofball** [comment]
@abcdefg30 also figured out the problem of getting dropped onto buildings and dying being an upstream as well, he said he was gonna make a PR for fixing that.

**Comment by Iamgoofball** [comment]
Also, yeah, the paradrop cameo is for generic paratroopers, airborne is for the US.

I'll look into making it load the proper cameo.

**Comment by Mailaender** [comment]
I fixed the remap in https://github.com/OpenRA/OpenRA/pull/10505.

**Comment by Mailaender** [comment]
Needs a rebase.

**Comment by Iamgoofball** [comment]
I'll fix the merge conflicts soon.

@abcdefg30 you still working on that fix for the paratrooper drop zone causing drops onto buildings?

**Comment by abcdefg30** [comment]
> @abcdefg30 you still working on that fix for the paratrooper drop zone causing drops onto buildings?

Yes, but I probably won't do more work until wednesday when I finished my exams.

**Comment by abcdefg30** [comment]
OpenRA/OpenRA#10505 was merged by the way.

**Comment by Mailaender** [comment]
No reaction for months now. Superseded by https://github.com/OpenRA/ra2/pull/167.

**Comment by abcdefg30** [comment]
> @abcdefg30 also figured out the problem of getting dropped onto buildings and dying being an upstream as well, he said he was gonna make a PR for fixing that.

FWIW: The pr is no here https://github.com/OpenRA/OpenRA/pull/11015.

**Pull request closed without merging**